static/js/pages/SitePage.tsx

```diff
@@ -46,6 +49,18 @@ export default function SitePage(): JSX.Element | null {
     }
   }
 
+  const onPublish = async () => {
+    if (publishIsPending) {
+      return
+    }
+    const response = await publishWebsite()
+    if (!response) {
+      return
+    } else {
+      // TBD
```

Comment:
What's to be determined here?

Reply:
What to show in the UI based on the API response (success or failure)